.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
@software-project is this intentional so that we can all debug with vscode? or should it be gitignored?

Reply:
yes, it's the config that allows to debug the tests in VS. But now I'm thinking maybe they should sit in some kind of docs, rather than here as not everyone might be using vscode. So I'm living it up to you.

Reply:
hm. I don't really mind either way. I know catherine at least uses vscode so maybe it's helpful. I'm fine with leaving it in here. Maybe you can open up a separate PR that adds some documentation for how to use this file if you're using vscode